gometalinter.json

```diff
@@ -1,12 +1,13 @@
 {
   "Vendor": true,
   "Deadline": "8m",
-  "Exclude": ["solver/pb/ops.pb.go", "api/services/control/control.pb.go"],
+  "Exclude": [".*.pb.go"],
```

Comment:
Formally you need to escape non-wildcard `.` as `\.`, but not mandatory now.

LGTM.

Reply:
Oh I see 😅